packages/cli/src/commands/deploy.ts

```diff
@@ -66,18 +70,24 @@ export const createDeployment = async (
             dryRun && Logger.info(pc.blue('Dry run: enabled'));
 
             const projectPath = process.cwd();
-            const { appId } = reactiveJson<AppManifest>(join(projectPath, 'manifest.json'));
+            const manifest = reactiveJson<AppManifest>(join(projectPath, 'manifest.json'));
 
             if (!noVerify) {
                 Logger.info('Performing type checks...');
                 await promiseExec('npx tsc --noEmit');
 
                 Logger.info('Performing eslint checks...');
                 await promiseExec('npx eslint src');
+
+                if (manifest.appType === 'platform-app') {
+                    Logger.info('Performing manifest checks...');
+                    const validManifest = await verifyManifest(manifest, platformAppManifestSchemaV1);
```

Comment:
Can we move it to line 73? Just after the manifest file read
```ts
const manifestContent = reactiveJson<AppManifest>(join(projectPath, 'manifest.json'));
const parsedManifest = manifest.appType === 'platform-app' ? await verifyManifest(manifest, platformAppManifestSchemaV1) : manifestContent
```

Reply:
cool thx, that makes it a lot easier.